db/00000000000007_task_aggregation_counters.up.sql

```diff
@@ -0,0 +1,14 @@
+-- Per-task report aggregation counters, used for metrics.
+--
+-- Fillfactor is lowered to improve the likelihood of heap-only tuple optimizations. See the
+-- discussion around this setting for the task_upload_counters table.
+CREATE TABLE task_aggregation_counters(
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,  -- artificial ID, internal only
+    task_id BIGINT NOT NULL,  -- task ID the counter is associated with
+    ord BIGINT NOT NULL,  -- the ordinal index of the task aggregation counter
+
+    success BIGINT NOT NULL DEFAULT 0,  -- reports successfully aggregated
```

Comment:
I see we're not going to track the other failures that could come up, like task upload counters do. Rationale? I'm nominally OK with this, since if we want this functionality it should be fairly trivial to bolt it on later.

Reply:
I'm implementing the least amount of functionality that gives us what we need; I'm OK leaning on operational metrics & logging for other issues, at least until we have a justification for splitting them out. (And yeah, I think it would be easy to add more to this later.)